google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -46,7 +46,7 @@
   <depends optional="true" config-file="app-engine-java-gradle.xml">org.jetbrains.plugins.gradle</depends>
   <depends optional="true" config-file="javaee-integration.xml">com.intellij.javaee</depends>
   <depends optional="true" config-file="gwt-integration.xml">com.intellij.gwt</depends>
-  <depends optional="true" config-file="google-cloud-apis.xml">org.jetbrains.idea.maven</depends>
+  <depends optional="true" config-file="google-cloud-maven.xml">org.jetbrains.idea.maven</depends>
```

Comment:
Was this intentional or we accidentally lost cloud API module dependency?

Reply:
ah I got it so this module intention is to be an umbrella module for all maven related sub-modules, correct? and this (for now) includes the whole cloud apis module?

Reply:
yes exactly (to your 2nd comment)